chromadb/test/property/test_persist.py

```diff
@@ -142,7 +143,7 @@ def test_sync_threshold(settings: Settings) -> None:
     )
 
     manager = system.instance(SegmentManager)
-    segment = manager.get_segment(collection.id, VectorReader)
+    segment = manager.get_segment(collection.id, LocalHnswSegment)
```

Comment:
curious why this has to be a more specific type now?

Reply:
I'm deprecating the `VectorReader` abstract class because it is only going to be used by local Chroma in the future. `MetadataReader` is deprecated as well.